libmscore/pitchspelling.cpp

```diff
@@ -684,11 +684,11 @@ void Score::spellNotelist(std::vector<Note*>& notes)
                         case 3:
                               k = end - start - 3;
                               changeAllTpcs(notes[end-3], tab[(notes[end-3]->pitch() % 12) * 2 + ((opt & (1<<k)) >> k)]);
-                              // FALLTHROUGH
+                              // Q_FALLTHROUGH();
```

Comment:
No, not as a comment, you need to drop the `// ` here

Reply:
just for `case 3:` or should I also modify `case 2:` this way as well?

Reply:
both in this file

Reply:
Done. e2ca31e